backend/inspirehep/records/marshmallow/literature/utils.py

```diff
@@ -5,8 +5,16 @@
 # inspirehep is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
 
+import re
+
+from pylatexenc.latexencode import UnicodeToLatexEncoder
+
 from inspirehep.records.api import InspireRecord
 
+# The regex selects math delimited by ``$...$`` or ``\(...\)``
+# where the delimiters are not escaped
+MATH_EXPRESSION_REGEX = re.compile(r"((?<!\\)\$.*?(?<!\\)\$|(?<!\\)\\\(.*?(?<!\\)\\\))")
```

Comment:
this will basically do the reverse of what is intended for displaymath
`some text $$a_b$$ more text`
```
MATH_EXPRESSION_REGEX.findall('some text $$a_b$$ more text')
['$$', '$$']
```

Reply:
We shouldn't have displaymath anywhere except maybe in abstracts, but we don't expose those in BibTeX/LaTeX. So handling `$$` would make it more complex for no real benefit.